dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -0,0 +1,115 @@
+﻿#include "pch.h"
+#include "ToastNotificationManager.h"
+#include "Microsoft.Windows.ToastNotifications.ToastNotificationManager.g.cpp"
+#include <winrt/Windows.ApplicationModel.Core.h>
+#include <winrt/Windows.Storage.h>
+#include <winrt/base.h>
+#include "ToastActivationCallback.h"
+#include "externs.h"
+#include "ToastNotificationUtility.h"
+#include <frameworkudk/pushnotifications.h>
+
+static winrt::event<winrt::Windows::Foundation::EventHandler<winrt::Microsoft::Windows::ToastNotifications::ToastActivatedEventArgs>> g_toastHandlers;
+
+static int g_toastHandlerCount = 0;
+
+winrt::event<winrt::Windows::Foundation::EventHandler<winrt::Microsoft::Windows::ToastNotifications::ToastActivatedEventArgs>>& GetToastHandlers()
+{
+    return g_toastHandlers;
+}
+
+int& GetToastHandleCount()
+{
+    return g_toastHandlerCount;
```

Comment:
Not sure, but I do not find logic in the PR to change this value. I am wondering when and where this value will be changed.